src/main/java/org/folio/circulation/resources/CheckInProcessAdapter.java

```diff
@@ -102,4 +116,23 @@
 
     return completedFuture(succeeded(item));
   }
+
+  CompletableFuture<HttpResult<CheckInProcessRecords>> sendCheckInPatronNotice(CheckInProcessRecords records) {
+    if (records.getLoan() == null) {
+      return CompletableFuture.completedFuture(HttpResult.succeeded(records));
+    }
+    return patronNoticePolicyRepository.lookupNoticePolicy(records.getLoan())
+      .thenApply(r -> r.next(policy -> {
+        sendCheckOutPatronNoticeWhenPolicyFound(records, policy);
+        return HttpResult.succeeded(records);
+      }));
+  }
+
+  private void sendCheckOutPatronNoticeWhenPolicyFound(CheckInProcessRecords records, PatronNoticePolicy patronNoticePolicy) {
```

Comment:
This method states it is sending a check out notice, yet uses the check in type?
`patronNoticePolicy.lookupLoanNoticeDescriptor(NoticeEventType.CHECK_IN, NoticeTiming.UPON_AT);`

Reply:
Misspelling, sorry. Should be CheckIn.